src/barnote.ts

```diff
@@ -43,34 +53,31 @@ export class BarNote extends Note {
   }
 
   // Get and set the type of Bar note. `type` must be one of `Vex.Flow.Barline.type`.
-  getType() {
+  getType(): number {
```

Comment:
Can this import the enum from stavebarline and classify that as the type?

Reply:
done